geode-protobuf/src/main/java/org/apache/geode/protocol/protobuf/ProtobufOpsProcessor.java

```diff
@@ -49,6 +55,7 @@ public ProtobufOpsProcessor(SerializationService serializationService,
         result = operationContext.getOperationHandler().process(serializationService,
             operationContext.getFromRequest().apply(request), context);
       } else {
+        logger.info("Received unauthorized request");
```

Comment:
I would have very much liked to log the source IP here to enable SOC teams to easier detect attacks, but we couldn't find a acceptable way to get the IP.

Reply:
Are we still supposed to be logging security violations using the securityLogWriter?

Reply:
just talked to JinMei and apparently this isn't the case anymore.